sites/public/pages/account/application.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState, useContext } from "react"
+import {
+  ApiClientContext,
+  RequireLogin,
+  t,
+  UserContext,
+  FormCard,
+} from "@bloom-housing/ui-components"
+import Link from "next/link"
+import FormSummaryDetails from "../../src/forms/applications/FormSummaryDetails"
+import FormsLayout from "../../layouts/forms"
+import { Application, Listing } from "@bloom-housing/backend-core/types"
+import { useRouter } from "next/router"
+import moment from "moment"
+
+const dateSubmitted = (submissionDate: Date) => {
```

Comment:
This seems like it could be a useful "generic" helper. Maybe we can relocate to `ui-components`?

Reply:
Re-located :)